src/layout/layout.js

```diff
@@ -287,7 +326,8 @@
         var drawerButton = this.element_.querySelector('.' +
           this.CssClasses_.DRAWER_BTN);
         if (!drawerButton) {
-          drawerButton = document.createElement('div');
+          drawerButton = document.createElement('button');
```

Comment:
Sadly, changing this from a `div` now could be a BC Break. Since, developers could be expecting `div.mdl-layout__drawer-button` to exist for selectors.